ImprovedAnvil/lang/English/setup.tra

```diff
@@ -7615,4 +7616,466 @@ Armor Class: 4 (6 vs. piercing and missile)
 Requires:
  6 Strength
 
-Weight: 28~
\ No newline at end of file
+Weight: 28~
+
+@12080 = ~Weaken Undead~
+
+@12081 = ~Weaken Undead
+(Abjuration)
+
+Level: 5
+Range: 0 
+Duration: 6 rounds
+Casting Time: 5 
+Area of Effect: The caster
+Saving Throw: None
+
+This powerful spell creates waves of anti-negative energy that sweep outward from the caster. These waves disrupt any undead that attempt to attack the caster, reducing the physical damage they deal by 10% and decreasing their chance for critical hit by 5% for 7 seconds. There is one wave per round for the duration of the spell, which renews the effects, but does not stack with itself. All undead are affected with no Saving Throw, this spell bypasses Magic Resistance.~
+
+@12082 = ~Disrupt Undead (Necromancy)
+Level: 7
+Range: 15 feet
+Duration: Instantaneous
+Casting Time: 7
+Area of Effect: One Undead Creature
+Saving Throw: None
+
+By means of this powerful spell, the pries uses his divine powers to disturb the negative forces floating inside an undead creature. The spell bypasses the target's magic resistance (if any) and inflicts a huge amount of damage, which varies depending on the undead type:
+
+- Skeleton Lords (and lesser undead creatures): 10D6 +10
+- Skeleton Warlords: 8D6 +8
+- Skeleton Grandlords: 6D6 +6
+- Liches, Master Vampires and Vampire Lords: 5D6 +5~
+
+@12083 = ~Cause Serious Wounds
+(Necromancy)
+
+Level: 4
+Sphere: Healing 
+Range: Touch 
+Duration: Permanent 
+Casting Time: 2 
+Area of Effect: 1 creature 
+Saving Throw: None 
+
+This spell is the reverse of Cure Serious Wounds. Instead of healing the creature touched, it causes damage. On a successful touch from the priest, the spell inflicts 17 points of damage upon the target. The next successful attack that the priest makes will inflict this effect, however, <PRO_HESHE> has 1 turn to make the attack before the spell fizzles. This spell directly affects the nervous system of the touched enemy, bypassing Magic Resistance, though enemies resistant to magical damage will be able to withstand the pain better. Because of the nature of this spell, all non-living creatures, such as undead or constructs are immune to its effects.
```

Comment:
non-living creatures probably shouldn't have a nervous system

Reply:
That's why they're immune to the spell. The logic for that is hidden in "because of the nature of this spell"